### PR TITLE
Teach Me QISKit Award 2018 Top Submissions

### PR DESCRIPTION
**Pull request by rraymondhp**
Adding top submissions to the  tutorial set.

### CODE REVIEW

File: appendix/README.md [hunk 1]
```diff
@@ -10,4 +10,5 @@ The tutorials under this Appendix are classified into four categories:
 - [Advanced QISKit](advanced_qiskit) - contain tutorials explaining the functionalities of QISKit.
 - [More on Quantum Information Science](more_qis) - tutorials related to Quantum Information Science.
 - [Algorithms and Applications](algo_app) - tutorials on Quantum Algorithms and Applications.
+- [Teach Me QISKit Tutorial Award 2018](teachmeqiskit2018) - top submissions to Teach Me QISKit Tutorial Award 2018
```
Comment (jaygambetta) [inline appendix/README.md]: i think to match the other folders we use teach_me_qiskit_2018

File: appendix/teachmeqiskit2018/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+## Teach Me QISKit Award 2018: Top Submissions
+
+This folder contains the top submissions of the Teach Me QISKit Award 2018.
+Most of the tutorials were written and tested with qiskit-sdk-py version 0.4.15.
```
Comment (jaygambetta) [inline appendix/teachmeqiskit2018/README.md]: qiskit-core as the repo name is going and the tag is still in qiskit-core

File: appendix/teachmeqiskit2018/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+Here is the list of the top submissions (in alphabetical order).
+
+- [CHSH Game](chsh_game/CHSH%20game-tutorial.ipynb) - play a quantum game.
```
Comment (jaygambetta) [inline appendix/teachmeqiskit2018/README.md]: could we give [name](link) by Awesome Quantum - some quick description

File: appendix/teachmeqiskit2018/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+- [Cryptography with BB84](cryptography/Cryptography.ipynb) - implementation of the BB84 protocol.
+- [E91 Quantum Key Distribution Protocol](e91_qkd/e91_quantum_key_distribution_protocol.ipynb) - a tutorial on quantum key distribution based on the protocol by Ekert in 1991.
+- [Elementary Arithmetic Operations](elementary_arithmetic_operations/elementary_arithmetic_operations.ipynb) - addition and modular exponentiation with quantum circuits.
+- [Exact Ising Model Simulation](exact_ising_model_simulation/Ising_time_evolution.ipynb) - how to simulate time evolution at zero time.
```
Comment (jaygambetta) [inline appendix/teachmeqiskit2018/README.md]: could we have winner tag on this one

**Comment by jaygambetta** [comment]
I forgot how good the submissions we got are. It is amazing to see this level of tutorials.

File: appendix/teachmeqiskit2018/README.md (hunk 2 above)
Comment (rraymondhp) [reply appendix/teachmeqiskit2018/README.md]: Done

File: appendix/teachmeqiskit2018/README.md (hunk 4 above)
Comment (rraymondhp) [reply appendix/teachmeqiskit2018/README.md]: Done

File: appendix/teachmeqiskit2018/README.md (hunk 3 above)
Comment (rraymondhp) [reply appendix/teachmeqiskit2018/README.md]: The link you gave is broken and I cannot find it. I modified the name and its description like below:
Awesome Quantum - get used to the quantum weirdness by playing a quantum game.

Comment (rraymondhp) [reply appendix/teachmeqiskit2018/README.md]: Done?

**Comment by qubeat** [comment]
I got an error trying to sign: This page (https://cla-assistant.io/accept/QISKit/qiskit-tutorial?pullRequest=213) is currently offline. ...

**Comment by jaygambetta** [comment]
@qubeat to sign the cla use https://cla-assistant.io/QISKit/qiskit-tutorial

**Comment by qubeat** [comment]
Is it possible to use in my tutorials
qiskit-tutorial/appendix/teachmeqiskit2018/state_distribution_in_qubit_chains/ 
an updated version with separate image directory (instead of embedded and not visible now by github viewer) and some minor changes? I already discussed that earlier in community. Currently it may be checked at 
https://github.com/qubeat/quchain/tree/master/IBM-QE/qiskit-simulator

**Comment by rraymondhp** [comment]
@qubeat  Thank you very much! 
There are two ways to do that. 
1. You can fork my repository, work on a branch, replace your notebooks with the new ones and merge with my repository. I will take care of the rest.
2. You can wait until this PR is being merged. After that, you can fork it and replace the new ones and issue a PR to have the new notebooks in place. 

What do you think?

**Comment by qubeat** [comment]
@rraymondhp Thank you. I'd better wait to do not interrupt your work.

File: appendix/teachmeqiskit2018/README.md (hunk 3 above)
Comment (rraymondhp) [reply appendix/teachmeqiskit2018/README.md]: Done.

**Pull request merged**